core/src/test/java/org/hibernate/ogm/backendtck/queries/SimpleQueriesTest.java

```diff
@@ -504,15 +498,17 @@ public void testEntitiesInsertedInCurrentSessionAreFoundByQueriesNotBasedOnHiber
 		hypothesis.setDescription( "In the morning it's darker than outside" );
 		hypothesis.setPosition( 30 );
 		session.persist( hypothesis );
-
-		if ( TestHelper.getCurrentDialectType().supportsQueries() ) {
-			assertQuery( session, query, 1, "Newly inserted entity should have been flushed and returned by the query" );
+		try {
```

Comment:
for  add this:
finally {
			session.delete( hypothesis );
		}
Athother, if exception occurs, entity not deleted. And other tests are failed